src/components/Breadcrumb/index.tsx

```diff
@@ -150,15 +152,15 @@ const VersionBreadcrumb: React.FC<VersionBreadcrumbProps> = ({
           {isBeta() ? (
             <StyledBreadcrumbLink
               data-cy="bc-waterfall"
-              to={getCommitsRoute(project)}
+              to={getCommitsRoute(projectIdentifier)}
               onClick={() =>
                 analytics.sendEvent({
                   name: "Click Link",
                   link: "waterfall",
                 })
               }
             >
-              {project}
+              {projectIdentifier}
```

Comment:
Should we treat project patches the same way?

Reply:
The project patches breadcrumb label is adjusted on line 176. I made a ticket to allow project patches to accept a projectIdentifier in the url like we do for the waterfall. https://jira.mongodb.org/browse/EVG-16590